crates/autopilot/src/run.rs

```diff
@@ -79,6 +79,12 @@ impl LivenessChecking for Liveness {
     }
 }
 
+impl Liveness {
+    pub fn auction(&self) {
+        *self.last_auction_time.write().unwrap() = Instant::now();
```

Comment:
nit: `expect` instead of `unwrap` with explanation of why we are impossible to run into this case.

Reply:
This is the same as `mutex.lock().unwrap()` which we commonly do throughout the code base so I think we don't need to start using `.expect()` here.
The reasoning is that an error here indicates an unknown state of the locked variable after a panic while holding the lock. There isn't really a good strategy to recover the "correct" value after that so we think panicking is fine on those errors over all.